build-system-abstraction.rst

```diff
@@ -0,0 +1,360 @@
+:PEP: XX
+:Title: Build system abstraction for pip/conda etc
+:Version: $Revision$
+:Last-Modified: $Date$
+:Author: Robert Collins <rbtcollins@hp.com>,
+         Nathaniel Smith <njs@pobox.com>
+:Discussions-To: distutils-sig <distutils-sig@python.org>
+:Status: Draft
+:Type: Standards Track
+:Content-Type: text/x-rst
+:Created: 26-Oct-2015
+:Post-History: XX
+:Requires: PEP-426
+
+
+Abstract
+========
+
+This PEP specifies a programmatic interface for pip [#pip]_ and other
+distribution or installation tools to use when working with Python
+source trees (both the developer tree - e.g. the git tree - and source
+distributions).
+
+The programmatic interface allows decoupling of pip from its current
+hard dependency on setuptools [#setuptools]_ able for two
+key reasons:
+
+1. It enables new boutique build systems that may be much easier to use
+   without requiring them to even appear to be setuptools.
+
+2. It facilitates setuptools itself changing it's user interface without
+   breaking pip, giving looser coupling.
+
+The programmatic interface also enables pip to install build time requirements
+for packages which is an important step in getting pip to full feature parity
+with the instalation components of easy-install.
```

Comment:
I thought the thing is also named with an underscore. At least, that's how it's used on https://pypi.python.org/pypi/setuptools